### ISSUE TITLE
set default jitsi configs per environment variable

### ISSUE THREAD

**Issue by jonnytest1** [thecodingmachine/workadventure#1077]
the frontend should be able to receive an evnironment variable that can overwrite the jitisINterfaceCOnfig and jitsiConfig to set global default parameters

**Comment by assel** [comment]
Yes please! (For example my beloved `TOOLBAR_BUTTONS` and `etherpad_base`. It could be defined as a property for the map.

**Comment by jonnytest1** [comment]
> Yes please! (For example my beloved `TOOLBAR_BUTTONS` and `etherpad_base`. It could be defined as a property for the map.

have you tried setting the properties as a map property ? i think it might work

**Comment by assel** [comment]
I just tried it and it doesn't work.
